cookbooks/bach_opentsdb/templates/default/etc_default_opentsdb.erb

```diff
@@ -0,0 +1,18 @@
+###############################################
+##   WARNING - THIS FILE IS MANAGED BY CHEF  ##
+## DO NOT MODIFY BY HAND, OR YOU'LL BE SORRY ##
+###############################################
+
+<% if node['bach_opentsdb']['tsd_user'] -%>
+TSD_USER=<%= node['bach_opentsdb']['tsd_user'] %>
+<% end -%>
+<% if node['bach_opentsdb']['tsd_group'] -%>
+TSD_GROUP=<%= node['bach_opentsdb']['tsd_group'] %>
+<% end -%>
+<% if node['bach_opentsdb']['daemon_opts'] -%>
+DAEMON_OPTS="<%= node['bach_opentsdb']['daemon_opts'] %>"
+<% end -%>
+<% if node['bach_opentsdb']['max_open_files'] -%>
+MAX_OPEN_FILES=<%= node['bach_opentsdb']['max_open_files'] %>
+<% end -%>
+JAVA_HOME=<%= node['bcpc']['hadoop']['java'] %>
```

Comment:
OpenTSDB shouldn't depend on BCPC per our CONTRIBUTING.md, please wrote a wrapper recipe to live in `bcpc_hadoop` which will apply the bcpc variables to the attributes made available by the bach_opentsdb cookbook (E.g.see the HDFS DU wrappers or for here something in `recipe [bcpc_hadoop::open_tsdb]` being `node.force_default['bach_opentsdb']['java_home'] = node ['bcpc']['hadoop']['java']`.)

You will want to ensire all code in `bach_opemtsdb` is independent of the other cookbooks.